spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/liquibase/LiquibaseProperties.java

```diff
@@ -53,6 +53,26 @@
 	 */
 	private String defaultSchema;
 
+	/**
+	 * Liquibase database schema.
+	 */
+	private String liquibaseSchema;
+
+	/**
+	 * Liquibase tablespace.
+	 */
+	private String liquibaseTablespace;
+
+	/**
+	 * Database changelog table.
```

Comment:
table name?

Reply:
What do you mean? I am using the same that used in liquibase project

Reply:
I mean that the description can be more informative.

Reply:
In `GlobalConfiguration` the description is 

> Name of table to use for tracking change history

which is way more informative than the Javadoc. I prefer to use that